edit-event-boundary/FormAtoms.test.ts

```diff
@@ -23,7 +23,7 @@ describe("EventFormValues tests", () => {
       placemark: undefined
     })
     expect(store.get(eventEditAtom)).toEqual({
-      ...DEFAULT_EDIT_EVENT_FORM_VALUES,
+      ..._DEFAULT_FORM_VALUES,
```

Comment:
This looks like an update on naming, since there might be another thing to edit, though when I looked up `DEFAULT_FORM_VALUES`, it still primarily looks like it'd be for editEvent.
Where else might we want to use this?

Reply:
This is an internal variable for the tests because we now because getting the actual default values for the edit form comes from calling the `defaultEditEventFormValues` function. Using a function allows us to always use the current date when we create the form, rather than only computing the current date once when the app launches.